test/e2e/persistent_volumes.go

```diff
@@ -749,6 +853,33 @@ func verifyGCEDiskAttached(diskName string, nodeName types.NodeName) bool {
 	return isAttached
 }
 
+// Sanity check for vSphere testing.  Verify the persistent disk attached to the node.
+func verifyVsphereDiskAttached(volumePath string, nodeName types.NodeName) bool {
```

Comment:
Actually returned result is not checked here. let me fix it.
We can move this check after POD creation, to make sure disk is attached to the node.